### PR TITLE
Add vl53l0cxv0dh stm32wb5mm dk

### PR DESCRIPTION
**Pull request by javad123javad**
Add device tree node for `VL53L0CXV0DH/1` and `ISM330DHCX` sensors

### CODE REVIEW

**Review by erwango — DISMISSED** [review]

**Review by nordicjm — DISMISSED** [review]
Needs converting to hwmv2

**Comment by javad123javad** [comment]
> Needs converting to hwmv2

Thanks,
Should I open a new PR?

**Comment by nordicjm** [comment]
> > Needs converting to hwmv2
> 
> Thanks, Should I open a new PR?

No, update this one

**Review by erwango — DISMISSED** [review]

File: boards/st/stm32wb5mm_dk/stm32wb5mm_dk.dts [hunk 1]
```diff
@@ -136,6 +137,27 @@ zephyr_udc0: &usb {
...
+		reg = <0x29>;
+		xshut-gpios = <&gpioc 6 GPIO_ACTIVE_LOW>;
+	};
+
+	ism330dhcx: ism330dhcx@6b {
+		compatible = "st,ism330dhcx";
+		reg = <0x6b>;
+		drdy-gpios = <&gpiod 2 GPIO_ACTIVE_HIGH>;
+		accel-odr=<3>;
+		gyro-odr=<2>;
```
Comment (GeorgeCGV) [inline boards/st/stm32wb5mm_dk/stm32wb5mm_dk.dts:157]:
```suggestion
		accel-odr = <ISM330DHCX_DT_ODR_52Hz>;
		gyro-odr = <ISM330DHCX_DT_ODR_26H>;
```

**Review by GeorgeCGV — DISMISSED** [review]

**Review by erwango — DISMISSED** [review]

**Comment by kartben** [comment]
rebased to resolve merge conflict

**Review by ajarmouni-st — APPROVED** [review]

**Comment by erwango** [comment]
@GeorgeCGV PTAL

**Review by kartben — APPROVED** [review]

**Review by erwango — APPROVED** [review]

**Pull request merged**